config/ucx.conf

```diff
@@ -9,6 +9,9 @@ UCX_GDR_COPY_BW=0MBs,get_dedicated:30GBs,put_dedicated:30GBs
 UCX_GDR_COPY_LAT=200ns
 UCX_DISTANCE_BW=auto,sys:16500MBs
 
+UCX_CUDA_IPC_LAT=2.5e-6
+UCX_CUDA_IPC_OVH=3.3e-6
```

Comment:
UCX_CUDA_IPC_OVERHEAD
to align with other vars, i.e ib_face.c, mm_iface.c

Reply:
Done